src/jemalloc.c

```diff
@@ -1361,6 +1361,10 @@ malloc_conf_init_helper(sc_data_t *sc_data, unsigned bin_shard_sizes[SC_NBINS],
 			    "muzzy_decay_ms", -1, NSTIME_SEC_MAX * KQU(1000) <
 			    QU(SSIZE_MAX) ? NSTIME_SEC_MAX * KQU(1000) :
 			    SSIZE_MAX);
+			CONF_HANDLE_SIZE_T(opt_process_madvise_max_batch,
+			    "max_batched_size", 0, SIZE_T_MAX,
+			    CONF_DONT_CHECK_MIN, CONF_DONT_CHECK_MAX,
```

Comment:
If `max_stack` is limiting the real max batch, why don't we set it as a limit here and clip it? It is misleading for users to read a fake `max_batched_size` when it is larger than 128.

Reply:
Good point. Added cap and clip.